app/views/businesses/new.html.erb

```diff
@@ -1,2 +1,16 @@
 <h1>Businesses#new</h1>
 <p>Find me in app/views/businesses/new.html.erb</p>
+
+<%= simple_form_for @business do |f| %>
+  <%= f.input :name, placeholder: 'Name of Your Business', error: 'Name is mandatory, please specify one' %>
+  <%= f.input :address, placeholder: 'Address of Your Business' %>
+  <%= f.input :instagram, placeholder: 'Instagram Profile Link' %>
+  <%= f.input :website, placeholder: 'Your Website url' %>
+  <%= f.input :photo, as: :file %>
```

Comment:
Please add `required` to all the field that validate existence in the DB. This is one filter that won't allow the user to submit a form that is not in the right shape, otherwise they'll get an error from the server.

Reply:
```
Required fields are marked with an * prepended to their labels.

By default all inputs are required. When the form object includes ActiveModel::Validations (which, for example, happens with Active Record models), fields are required only when there is presence validation. Otherwise, Simple Form will mark fields as optional. For performance reasons, this detection is skipped on validations that make use of conditional options, such as :if and :unless.

And of course, the required property of any input can be overwritten as needed:

<%= simple_form_for @user do |f| %>
  <%= f.input :name, required: false %>
  <%= f.input :username %>
  <%= f.input :password %>
  <%= f.button :submit %>
<% end %>
```